src/lib/hooks/usePaymentToken.ts

```diff
@@ -0,0 +1,22 @@
+import { getDomainId } from '../util/domains/domains';
+import { usePaymentTokenForDomain } from './usePaymentTokenForDomain';
+import { usePaymentTokenInfo } from './usePaymentTokenInfo';
+
+export const usePaymentToken = (zna: string) => {
```

Comment:
@domw30 I set the param as `zna` rather than `domainId`, as I think `zna` provides a more readable interface and abstracts the conversion from ZNA -> ID. I.e. `use payment token for wilder.wheels` rather than `use payment token for 0x...`. Not sure if that's a pattern you'd like to keep or not - if not I'll revert it to `domainId`.

Reply:
Yeah sweet good idea, reads well to me!